app/views/static_pages/home/_recent_products.html.erb

```diff
@@ -4,87 +4,35 @@
-                    </div>
-                                                    </span>
-                    <p>Lorem ipsum dolor sit amet, consectetuer dolorado adipiscing elit diam, sed diam nonumy
-                      nicely done...</p>
-                  </div>
-                </article>
-                <article class="entry-item">
-                  <div class="entry-thumb spa-hover-effect-2">
-                    <img src="http://placehold.it/370x280" alt="">
-                    <div class="spa-wrap-thumb-add-feature">
-                      <a href="#" class="fa fa-heart-o"></a>
-                      <a href="#" class="fa fa-shopping-basket"></a>
-                      <a href="#" class="fa fa-search"></a>
-                    </div>
-                  </div>
-                  <div class="entry-content woocommerce">
-                    <div class="spa-wrap-line">
-                      <h4 class="entry-title"><a href="#">MERCEDES M3 GTR</a></h4>
-                    </div>
-                    <p>Lorem ipsum dolor sit amet, consectetuer dolorado adipiscing elit diam, sed diam nonumy
-                      nicely done...</p>
-                  </div>
-                </article>
-                <article class="entry-item">
-                  <div class="entry-thumb spa-hover-effect-2">
-                    <img src="http://placehold.it/370x280" alt="">
-                    <div class="spa-wrap-thumb-add-feature">
-                      <a href="#" class="fa fa-heart-o"></a>
-                      <a href="#" class="fa fa-shopping-basket"></a>
-                      <a href="#" class="fa fa-search"></a>
-                    </div>
-                  </div>
-                  <div class="entry-content woocommerce">
-                    <div class="spa-wrap-line">
-                      <h4 class="entry-title"><a href="#">MERCEDES M3 GTR</a></h4>
-                    </div>
-                    <p>Lorem ipsum dolor sit amet, consectetuer dolorado adipiscing elit diam, sed diam nonumy
-                      nicely done...</p>
-                  </div>
-                </article>
+                <% @projects.each do |p| %>
+                    <article class="entry-item">
+                      <div class="entry-thumb spa-hover-effect-2">
+                        <div class="rotate" style="display:none;">
+                          <% p.resources.each do |r| %>
+                              <%= link_to((image_tag r.image.url), project_path(p)) %>
+                          <% end %>
+                          <span class="spa-sale">Otwórz</span>
+                        </div>
+                        <div class="entry-content woocommerce">
+                          <div class="spa-wrap-line">
+                            <h4 class="entry-title"><%= link_to p.title, project_path(p) %></h4>
+                          </div>
+                        </div>
+                        <p><%= p.description %></p>
```

Comment:
move this tag inside `entry-content woocommerce` div, right after `spa-wrap-line` div